SU-SOE/stanford_soe_helper
# 7.x 2.x theming bugs

# READY FOR REVIEW

# Summary
- Theming bugs epic

# Needed By (Date)
- Sprint end

# Criticality
- Fixes broken stuff


# Steps to Test
- Switch to this branch
- May need to revert feature
- See ticket tasks for testing: https://stanfordits.atlassian.net/browse/SOE-2443

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2443

## Related PRs

## More Information

## Folks to notify
@kerri-augenstein


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
